lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -960,9 +972,26 @@ def get_sort_order(ascending):
                 return "-"
 
     def check_is_admin(self):
-
+        """
+        Check Auth for your admin role and see if it is an allowed admin role
+        :return:
+        """
         return int(self._is_admin(self.token))
 
+    def get_admin_permission(self):
+        """
+        Get your your type of admin permissions
+        :return:
+        """
+        aau = AdminAuthUtil(self.auth_url, self.admin_roles)
+        roles = list(aau._fetch_user_roles(self.token))
+        permission = None
+        if "EE2_ADMIN" in roles:
+            permission = "w"
```

Comment:
ok. But spec says permission is `One of ('read' | 'write' | 'none')`. Just in case people will complain, we should update either one.

Reply:
Fixed, thanks